lib/converter.js

```diff
@@ -1,10 +1,28 @@
 /* global Opal */
 const util = require('util')
-const fs = require('fs')
+const {writeFile, readFile} = require('fs').promises
 const path = require('path')
-const writeFile = util.promisify(fs.writeFile)
 const puppeteer = require('puppeteer')
 const asciidoctor = require('asciidoctor.js')()
+const getPort = require('get-port')
+const handler = require('serve-handler')
+const {createServer} = require('http')
+const debug = util.debuglog('asciidoctor-pdf')
+
+const STATIC_ASSETS = {
+  '/asciidoctor.css': {
+    src: path.join(__dirname, '..', 'examples', 'document', 'asciidoctor.css'),
+    type: 'text/css',
+  },
+  '/asciidoctor-document.css': {
```

Comment:
It should be `/document.css` to be consistent with `<link href="document.css" rel="stylesheet">` right ?

Reply:
Yes.

I was wary of naming collisions at this point and never completed the thinking around it.

Also, it made me think of the `stylesheet` and `linkcss` well known Asciidoctor attributes.

I'm happy to change either of these in a way which is consistent with Asciidoctor behaviour.